Src/StackifyLib/Internal/Auth/Claims/AppClaimsManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+
+namespace StackifyLib.Internal.Auth.Claims
+{
+    internal static class AppClaimsManager
+    {
+        private readonly static IAppClaimsBuilder _claimsBuilder = AppClaimsBuilderFactory.Get();
+
+        private static AppClaims _claims = null;
+
+        public static AppClaims Get()
+        {
+            if (_claims != null)
+                return _claims;
+
+            GetAsync().Wait();
```

Comment:
I really don't like this and would like to find a way to ensure this is handled asynchronously...